hal/Cargo.toml

```diff
@@ -163,3 +163,7 @@ samv71q20b = ["atsamv71q20b", "v71", "pins-144", "flash-1M", "device-selected"]
 samv71q20b-rt = ["samv71q20b", "atsamv71q20b/rt"]
 samv71q21b = ["atsamv71q21b", "v71", "pins-144", "flash-2M", "device-selected"]
 samv71q21b-rt = ["samv71q21b", "atsamv71q21b/rt"]
+
+[package.metadata.docs.rs]
+# enable all possible features for docs.rs
+features = ["samv71q21b", "samv71q21b-rt", "unproven", "usart-spi-host-without-select", "reconfigurable-system-pins"]
```

Comment:
I believe you don't need `samv71q21b` when you also have `samv71q21b-rt`.

Thanks for the pull request, I had been meaning to eventually get around to fixing this.

Reply:
All set, I removed those.

(I just had it there from copying the samd crate, which lists `["samd21g", "samd21g-rt", "unproven", "usb"]`)